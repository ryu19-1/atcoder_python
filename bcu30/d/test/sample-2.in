8 10
-499 -120 32 255 571 890 1011 1256
0
-200
2500
364
-117
50
-612
889
32
364
