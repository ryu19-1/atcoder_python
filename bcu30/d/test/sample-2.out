4634
5594
16604
4060
5102
4470
8292
4696
4506
4060
